llvm/lib/Target/SystemZ/SystemZISelLowering.cpp

```diff
@@ -8096,6 +8096,24 @@ static void createPHIsForSelects(SmallVector<MachineInstr*, 8> &Selects,
   MF->getProperties().reset(MachineFunctionProperties::Property::NoPHIs);
 }
 
+MachineBasicBlock *
+SystemZTargetLowering::emitAdjCallStack(MachineInstr &MI,
+                                        MachineBasicBlock *BB) const {
+  // Do the work of MachineFrameInfo::computeMaxCallFrameSize() early and
+  // remove these nodes. Given that these nodes start out as a glued sequence
+  // it seems best to remove them here after instruction selection and
+  // scheduling.  NB: MIR testing does not work (yet) for call frames with
+  // this.
```

Comment:
Can you elaborate what that problem with MIR testing is, and whether we need to fix it before landing this upstream?

Reply:
wow - now that I checked it turns out that MIR actually *does* save this info as part of the .mir file. So this 'NB' should just be removed. Sorry - I thought I was in new territory here...